deploy/cdk/lib/cdk-pipeline-deploy.ts

```diff
@@ -95,11 +95,11 @@ export class CDKPipelineDeploy extends Construct {
         phases: {
           install: {
             commands: [
+              'n stable',
```

Comment:
Does this mean that every time we redeploy this, it will install the latest stable version of Node?